pandas/core/indexes/base.py

```diff
@@ -5135,9 +5135,30 @@ def delete(self, loc):
         """
         Make new Index with passed location(-s) deleted.
 
+        Parameters
+        ----------
+        loc : int or list of int
+            Location of item(-s) which will be deleted.
+            Use list of integer as loc parameter to delete multiple locations.
```

Comment:
What do you think about the next?

Use a list of locations to delete more than one value at the same time

Reply:
looks good